actioner/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "22996056-6abc-4d33-be11-3e27626b8a26",
+  "app_id": "actioner",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Alternative way to connect your Datadog account with Actioner",
+    "title": "actioner",
+    "media": [],
```

Comment:
Please update this section with the images you include in the media carousel as mentioned [here](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file)